includes/functions-ur-page.php

```diff
@@ -150,6 +150,13 @@ function ur_nav_menu_items( $items ) {
 			}
 		}
 	}
+	$customer_logout = get_option( 'user_registration_logout_endpoint', 'user-logout' );
+
+	foreach( $items as $item ) {
+    if( $item->title == "Logout" && $customer_logout && 'yes' === get_option( 'user_registration_disable_logout_confirmation', 'no' ) ) {
```

Comment:
Here `$item->title == "Logout"` should not be included in the condition as the menu title could be changed by the user.

Reply:
The issue has been solved